.docker/scripts/init.sh

```diff
@@ -5,11 +5,15 @@
 set -e
 
 CKAN_USER_NAME="${CKAN_USER_NAME:-admin}"
-CKAN_USER_PASSWORD="${CKAN_USER_PASSWORD:-password}"
+CKAN_USER_PASSWORD="${CKAN_USER_PASSWORD:-Password123!}"
```

Comment:
Since this is a shell script, we could have this be a dynamic password that changes on each run. (since the system should be destroyed after running)